### PR TITLE
Adopt Better-Less grammar

### LINKED ISSUE #171239: Consider adopting a new Less grammar
Now that https://github.com/atom/language-less has been archived we need to find a new Less grammar. 

https://github.com/radium-v/Better-Less has some promise, but it hasn't been updated since 2018. I'm reaching out via an issue to see if the owner has interest in reviving the project.

### PR DESCRIPTION
**Pull request by alexr00**
Fixes #171239

### CODE REVIEW

**Comment by aeschli** [comment]
Looks good to me!

**Comment by alexr00** [comment]
@aeschli great, thanks for taking a look!

**Comment by alexr00** [comment]
@radium-v, just a heads up that this is going in! I'll keep it in VS Code insiders until ~February.

File: extensions/theme-defaults/themes/dark_vs.json [hunk 1]
```diff
@@ -308,7 +312,7 @@
 				"support.type.property-name",
 				"variable.css",
 				"variable.scss",
-				"variable.other.less",
+				"source.css.less variable.other",
```
Comment (aeschli) [inline extensions/theme-defaults/themes/dark_vs.json:315]: I beliebe we can now merge the 3 lines with:
```suggestion
				"variable.css",
				"variable.scss",
				"source.css.less variable.other",
```

**Review by aeschli — APPROVED** [review]

File: extensions/theme-defaults/themes/dark_vs.json (hunk 1 above)
Comment (aeschli) [reply extensions/theme-defaults/themes/dark_vs.json:315]: I created https://github.com/microsoft/vscode/pull/197657 for that change

**Pull request merged**